packages/core/src/app/order/OrderSummaryTotal.spec.tsx

```diff
@@ -19,13 +21,15 @@ describe('OrderSummaryTotal', () => {
     describe('when shopper has same currency as store', () => {
         beforeEach(() => {
             orderSummaryTotal = mount(
-                <LocaleContext.Provider value={localeContext}>
-                    <OrderSummaryTotal
-                        orderAmount={100}
-                        shopperCurrencyCode="USD"
-                        storeCurrencyCode="USD"
-                    />
-                </LocaleContext.Provider>,
+                <CheckoutProvider checkoutService={createCheckoutService()}>
```

Comment:
Thanks for updating the testWrapper.
This line and L53 appear to be overlooked.

Reply:
Hm,  but I think we should use new render only for new test with react testing library and not for enzyme?